seata-spring-boot-starter/src/main/java/io/seata/spring/boot/autoconfigure/properties/registry/RegistryEurekaProperties.java

```diff
@@ -0,0 +1,60 @@
+/*
+ *  Copyright 1999-2019 Seata.io Group.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package io.seata.spring.boot.autoconfigure.properties.registry;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+import static io.seata.spring.boot.autoconfigure.StarterConstants.REGISTRY_EUREKA_PREFIX;
+
+/**
+ * @author xingfudeshi@gmail.com
+ * @date 2019/10/03
+ */
+@Component
+@ConfigurationProperties(prefix = REGISTRY_EUREKA_PREFIX)
+public class RegistryEurekaProperties {
+    private String serviceUrl = "http://localhost:1001/eureka";
```

Comment:
commonly used port 8761, Do you see if it is necessary to modify.

Reply:
I refer to the configurations in this file https://github.com/seata/seata/blob/develop/config/seata-config-core/src/main/resources/registry.conf.
Seems some configurations are not unified in Seata.I'll modify it.Thanks!

Reply:
done.